documentation/src/test/java/org/hibernate/validator/referenceguide/chapter11/context/MyFutureValidator.java

```diff
@@ -12,14 +10,14 @@
 import org.hibernate.validator.constraintvalidation.HibernateConstraintValidatorContext;
 
 //tag::include[]
-public class MyFutureValidator implements ConstraintValidator<Future, Date> {
+public class MyFutureValidator implements ConstraintValidator<Future, Instant> {
```

Comment:
I thought it was a good idea for our examples to use the new java.time API. Nothing more.